### DISCUSSION TITLE
SvelteKit with Deno?

### DISCUSSION THREAD

**Discussion by ioucyf** [sveltejs/kit#7308] [Q&A]
Is it possible to run SvelteKit with Deno instead of Node?

**Comment by mustofa-id** [comment]
There is community adapter for Deno: https://github.com/pluvial/svelte-adapter-deno

**Reply by theArianit** [reply]
I think this is for deploying it on deno deploy

**Reply by samal-rasmussen** [reply]
Development on that seems to have stalled.

This one is actively developed instead: https://github.com/dbushell/sveltekit-adapter-deno

**Comment by benmccann** [comment]
See https://github.com/sveltejs/kit/issues/8324 for Deno dev mode

**Reply by ioucyf** [reply]
That's exactly what I meant. He spoke my mind so very well, and more. 

Thank you for letting me aware of his post to follow it, and use his workaround hack to start developing sveltekit in deno!
